src/main/resources/migrations/00018_rename_settlement_coumns.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE transaction rename COLUMN settlement_submitted_time to capture_submitted_date;
```

Comment:
Can we have this migration as separate PR or this might be opportunity to move it to `transaction_details` if it is not required at table level ?

Reply:
yeah, maybe we should move to details - I'll do that